output/2023-02-22.csv

```diff
@@ -0,0 +1,5 @@
+Metric,Value

```

Comment:
I think we should consider .gitignore the outputs of the program.

Reply:
+1